etl_manager/etl.py

```diff
@@ -402,7 +402,7 @@ def job_run_state(self) :
         state = 'no_run_state'
```

Comment:
As you're in here, I don't think the `state` variable is used so it can be removed (unless I'm missing something stupid here)